lib/adapters/sequelize.js

```diff
@@ -76,10 +76,11 @@ module.exports = Bone => {
     // EXISTS
     // static bulkCreate() {}
 
-    static async count(options = {}) {
+    static async count(options = {}, queryOptions = {}) {
       const { where, col, group } = options;
-      const spell = super.find(where);
+      let spell = super.find(where);
       if (Array.isArray(group)) spell.$group(...group);
+      if (queryOptions.paranoid === false || options.paranoid === false) spell = spell.unscoped;
```

Comment:
It seems there's no need to add an extra queryOptions. Sequelize treats options as the all in one options object, which is mostly handled by `translateOptions()` in our adapter.